config/locales/en.yml

```diff
@@ -1102,6 +1103,7 @@ en:
         details: Details
         address: Address
         products: Products
+        no_open_or_upcoming_order_cycle: "No Upcoming OC"
```

Comment:
imo, users don't necessarily get the acronym. I'd use "No Upcoming Order Cycle"

Reply:
I changed this already here, by the way: https://github.com/openfoodfoundation/openfoodnetwork/pull/2733/commits/c48cac20363f27f8688b71dda3c172e45b85aab2